explorer/interpreter/exec_program.cpp

```diff
@@ -21,57 +22,61 @@ namespace Carbon {
 auto AnalyzeProgram(Nonnull<Arena*> arena, AST ast,
                     Nonnull<TraceStream*> trace_stream,
                     Nonnull<llvm::raw_ostream*> print_stream) -> ErrorOr<AST> {
-  if (trace_stream->is_enabled()) {
-    *trace_stream << "********** source program **********\n";
-    for (int i = ast.num_prelude_declarations;
-         i < static_cast<int>(ast.declarations.size()); ++i) {
-      *trace_stream << *ast.declarations[i];
+  return RunWithStackSpace<ErrorOr<AST>>([&]() -> ErrorOr<AST> {
```

Comment:
Here, too, should we split out an `Impl` function to avoid the whitespace changes?

Reply:
Removing RunWithStackSpace from this file since ParseAndExecute is handling it. I think I'd added this early in implementation when I was still hashing out the approach.